napari/_qt/layers/qt_labels_layer.py

```diff
@@ -71,15 +73,15 @@ def __init__(self, layer):
         )
 
         # shuffle colormap button
-        self.colormapUpdate = QPushButton('shuffle colors')
+        self.colormapUpdate = QPushButton('shuffle\ncolors')
```

Comment:
I like making the label multiline, but the pushbutton is a little too tall now, see below - any chance you can make it same height as other icons 
![image](https://user-images.githubusercontent.com/6531703/83098522-c410e400-a05f-11ea-9475-37be1568ea43.png)

Reply:
oh also, can you make the line width of the eraser one thicker, see how it is a little thin compared to the other icons

Reply:
from team meeting - @tlambert03 is going to make shuffle color a shuffle icon and make the thickness of the eraser match the thickness of the others. thanks @tlambert03!!